packages/datatrak-web/src/features/Tasks/TasksTable/RepeatScheduleFilter.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { RRULE_FREQUENCIES } from '@tupaia/utils';
+import { TaskFilterType } from '../../../types';
+import { SelectFilter } from './SelectFilter';
+
+interface RepeatScheduleFilterProps {
+  onChange: (value: string) => void;
+  filter: { value: TaskFilterType } | undefined;
+}
+
+export const RepeatScheduleFilter = ({ onChange, filter }: RepeatScheduleFilterProps) => {
+  // if there is a selected status filter and it is not 'repeating', no options should show
+  const options = Object.entries(RRULE_FREQUENCIES).map(([key, value]) => ({
+    value,
+    label: `${key.charAt(0)}${key.slice(1).toLowerCase()}`,
```

Comment:
this looks like the same formatting as that other one I commented on. I think it's ok to just repeat it for now since it's just one time in the backend and one in the front end but if we use it again probably worth extracting out.